Add PiP CFR experiment for 78

ncloudioj opened the pull request:
Yes, this experiment is targeting Firefox *78*.

Experimenter ticket: https://experimenter.services.mozilla.com/experiments/message-router-content-experiment-high-velocity-message-testing-cfr-to-pip/

punamdahiya commented on messaging-experiments.json:
```diff
@@ -1146,5 +1146,488 @@
+    "filter_expression": "env.channel == 'release' && env.version >= '78.' && env.version < '79.'",
+    "targeting": "locale == 'en-US' && userPrefs.cfrFeatures &&\n'media.videocontrols.picture-in-picture.enabled'|preferenceValue &&\n[userId, 'messaging-experiments-cfr']|bucketSample(5312, 650, 10000) &&\n((currentDate|date - profileAgeCreated) / 86400000) > 7\n",
+    "arguments": {
+      "slug": "bug-1656382-message-recommend-picture-in-picture-cfr-release-77-80",
```
curious how we ensure slug above is same as slug from experimeneter.services.mozilla.com. I see that generated on click of Ready To Ship in Experimenter

ncloudioj replied:
No, we can't get the actual slug for now. This is merely a placeholder, and we have to update it once it's generated on Experimenter (and yes, that's very error-prone).

piatra commented on messaging-experiments.json:
```diff
@@ -1146,5 +1146,488 @@
+    "id": "MESSAGE_ROUTER_PICTURE_IN_PICTURE_CFR",
+    "enabled": true,
+    "filter_expression": "env.channel == 'release' && env.version >= '78.' && env.version < '79.'",
+    "targeting": "locale == 'en-US' && userPrefs.cfrFeatures &&\n'media.videocontrols.picture-in-picture.enabled'|preferenceValue &&\n[userId, 'messaging-experiments-cfr']|bucketSample(5312, 650, 10000) &&\n((currentDate|date - profileAgeCreated) / 86400000) > 7\n",
```
Doesn't exclude people that opted out of studies `app.shield.optoutstudies.enabled`

piatra commented on messaging-experiments.json:
```diff
@@ -1146,5 +1146,488 @@
+              "text": "Keep yourself in the loop. With Picture-in-Picture you can watch the latest news while you visit other websites or apps - hover over a video and select the blue icon."
+            },
+            "frequency": {
+              "lifetime": 30,
```
Is this correct? We want to show the message 30 times? The usual lifetime is `3`.

ncloudioj replied:
Confirmed with experiment owners that we're good to use the lifetime as 3.

piatra commented on messaging-experiments.json:
```diff
@@ -1146,5 +1146,488 @@
+          "value": {
+            "id": "PIP_CFR_INFORMATIONAL",
+            "groups": [
+              "cfr"
```
If we add the message to the `cfr` group then production messages can prevent experiment messages from being shown.

ncloudioj replied:
Switching over to "cfr-experiments".

ncloudioj commented:
@piatra FYI, the change highlights:

* Updated experiment slug
* Updated the branch slugs according to Experimenter
* Changed targeting version from 78 to 78+79
* Changed the sample size from 6.5% to 5%
* Added `platformName` to target Windows and Mac OS X

piatra commented:
🚢
